File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Dagpengegrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.DagpengeTypeCti;

import java.io.Serializable;

public class Dagpengegrunnlag implements Serializable {


    private int ar;
    private DagpengeTypeCti dagpengeType;
    private int uavkortetDagpengegrunnlag;
    private int utbetalteDagpenger;
    private int ferietillegg;
    private int barnetillegg;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/Dagpengegrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.DagpengetypeEnum
import java.io.Serializable

class Dagpengegrunnlag : Serializable {
    var ar = 0
    var dagpengetypeEnum: DagpengetypeEnum? = null
    var uavkortetDagpengegrunnlag = 0
    var utbetalteDagpenger = 0
    var ferietillegg = 0
    var barnetillegg = 0

}